ui/raidboss/emulator/data/Combatant.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+class Combatant {
+  constructor(id, name) {
+    this.id = id;
+    this.name = name.split('(')[0];
+    this.states = {};
+    this.significantStates = [];
+    this.latesttimestamp = null;
```

Comment:
I'm trying hard to not leave bikeshed comments about naming things, but ... maybe `latestTimeStamp`?

Reply:
This one's actually a mistake, it got caught in my renaming of `Timestamp` => `timestamp` :blush: